pkg/codegen/internal/test/testdata/cyclic-types/python/pulumi_example/pulumiplugin.json

```diff
@@ -0,0 +1,4 @@
+{
+  "resource": true,
+  "name": "example"
+}
```

Comment:
Nit: One minor thing that's bugged me about this emitted file is that we don't include a trailing newline. As part of emitting this by default now, would you mind fixing it to include the trailing newline? Similar to #7480.

Perhaps include the newline in the result returned from the `JSON()` method?